src/schema/v2/conversation/__tests__/conversation.test.js

```diff
@@ -611,6 +613,47 @@ describe("Me", () => {
             expect(inquiryRequest.questions).toHaveLength(2)
             expect(inquiryRequest.questions).toMatchSnapshot()
             expect(inquiryRequest.shippingLocation).toMatchSnapshot()
+            expect(inquiryRequest.formattedFirstMessage).toMatchSnapshot()
```

Comment:
Its totally cool to keep this since there's already snapshots in this file but its better to avoid them completely

Reply:
Makes sense! I'll merge this for now to unblock the FE work and will block some time on Friday to take out snapshot usage from this file 👍